ipatests/test_integration/test_adtrust_install.py

```diff
@@ -216,3 +220,88 @@ def test_schema_compat_attribute(self):
+        serverid = realm_to_serverid(self.master.domain.realm)
+        dirsrv_service = "dirsrv@%s.service" % serverid
+        console_msg = (
+            "Enabling plugin\n"
+            "This setting will not take effect until you restart Directory Server.\n"
+            "The rpcbind service may need to be started"
+        )
+        tasks.kinit_admin(self.master)
+        result = self.master.run_command(
+            ["ipa-nis-manage", "enable"],
+            stdin_text=self.master.config.admin_password,
+        )
+        self.master.run_command(["systemctl", "restart", dirsrv_service])
+        time.sleep(DIRSRV_SLEEP)
+        self.master.run_command(["systemctl", "restart", "rpcbind"])
+        assert console_msg in result.stdout_text
+
+    def test_ipa_nis_manage_disable(self):
+        """
+        This testcase checks if ipa-nis-manage disable
+        command disable plugin on an IPA Master
+        """
+        serverid = realm_to_serverid(self.master.domain.realm)
+        dirsrv_service = "dirsrv@%s.service" % serverid
+        msg = "This setting will not take effect until you restart Directory Server."
+        tasks.kinit_admin(self.master)
+        result = self.master.run_command(
+            ["ipa-nis-manage", "disable"],
+            stdin_text=self.master.config.admin_password,
+        )
+        assert msg in result.stdout_text
+        self.master.run_command(["systemctl", "restart", dirsrv_service])
+        time.sleep(DIRSRV_SLEEP)
+
+    def test_ipa_nis_manage_enable_incorrect_password(self):
+        """
+        This testcase checks if ipa-nis-manage enable
+        command throws error on console for invalid DS admin password
+        """
+        msg = "Insufficient access:  Invalid credentials\n"
+        result = self.master.run_command(
+            ["ipa-nis-manage", "enable"],
+            stdin_text='Invalid_pwd',
+            raiseonerr=False,
+        )
+        assert result.returncode == 1
+        assert msg in result.stderr_text
+
+    def test_ipa_nis_manage_no_map_message(self):
+        """
+        This testcase checks that when ipa-nis-manage is enabled
+        and search for user is done on compat tree, there
+        is no message as 'map_unlock: old way MAP_MONITOR_DISABLED'
+        in dirsrv error logs
+        """
+        serverid = realm_to_serverid(self.master.domain.realm)
+        dirsrv_service = "dirsrv@%s.service" % serverid
+        msg = "schema-compat - map_unlock: old way MAP_MONITOR_DISABLED"
+        tasks.kinit_admin(self.master)
+        result = self.master.run_command(
+            ["ipa-nis-manage", "enable"],
+            stdin_text=self.master.config.admin_password,
+        )
+        assert result.returncode == 0
+        self.master.run_command(["systemctl", "restart", dirsrv_service])
+        time.sleep(DIRSRV_SLEEP)
+        instance = realm_to_serverid(self.master.domain.realm)
+        log_path = paths.SLAPD_INSTANCE_ERROR_LOG_TEMPLATE % instance
```

Comment:
After moving the said code to `install()`, you can use:
`log_path = paths.SLAPD_INSTANCE_ERROR_LOG_TEMPLATE % self.serverid`